fff

```diff
@@ -368,6 +369,49 @@ trash() {
     fi
 }
 
+bulk_rename() {
+  local tmp_file="$(mktemp)"
```

Comment:
You can use this dir to store temp files: `${XDG_CACHE_HOME:=${HOME}/.cache}/fff`

Reply:
Would you please check this one? Not sure if I got you correctly